CONTRIBUTING.md

```diff
@@ -45,14 +45,19 @@ Our work is done directly on Github and PR's are sent to the github repo by core
 
 ## Setting up for local development
 
-This section should get you running with **Amplify Flutter** and get you familiar with the basics of the codebase. You will need the latest version of [nodejs](https://nodejs.org/en/) on your system and developing locally also requires `yarn` workspaces. You can install it [here](https://classic.yarnpkg.com/en/docs/install#mac-stable).
+This section should get you running with **Amplify Flutter** and get you familiar with the basics of the codebase.
 
 Start by, [Forking](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) the main branch of [amplify-flutter](https://github.com/aws-amplify/amplify-flutter).
 
+You will need to install `melos` for dependency management.
+Run `melos bootstrap --ignore="*example*"` to link local packages together and install remaining dependencies.
+See [invertase/melos](https://github.com/invertase/melos) for more instructions on how to use `melos`.
```

Comment:
Might be worth noting that running `pub get` is not required on any of the packages when using `melos bootstrap`.

I'd also recommend committing this to prevent auto `pub get` behaviour for Vscode users: https://github.com/FirebaseExtended/flutterfire/blob/master/.vscode/settings.json